tools/test-fwe.sh

```diff
@@ -5,16 +5,23 @@
 set -eo pipefail
 
 WITH_ROS2_SUPPORT="false"
+IGNORE_TESTS=""
```

Comment:
This should now be renamed to something like `EXTRA_CTEST_ARGS` otherwise it will be confusing.

Reply:
Done